lms/djangoapps/commerce/views.py

```diff
@@ -65,6 +67,11 @@ def checkout_receipt(request):
             "If your course does not appear on your dashboard, contact {payment_support_link}."
         ).format(payment_support_link=payment_support_link)
 
+    commerce_configuration = CommerceConfiguration.current()
+    if is_payment_complete and commerce_configuration.enabled and commerce_configuration.is_cache_enabled:
+        cache_key = commerce_configuration.CACHE_KEY + '.' + request.user.username
+        cache.delete(cache_key)
```

Comment:
the idea is that whenever a new order is placed we should clear cache and get the updated data from otto.